packages/react-magma-dom/src/components/StyledButton/index.tsx

```diff
@@ -17,11 +17,13 @@ import {
   buildFocusColor,
 } from './styles';
 import { ThemeContext } from '../../theme/ThemeContext';
-import { ButtonProps, ButtonType } from '../Button';
+import { ButtonType, ButtonProps, ButtonSize, ButtonVariant } from '../Button';
+import { Spinner } from '../Spinner';
 
-interface StyledButtonProps extends ButtonProps {
+export interface StyledButtonProps extends ButtonProps {
   href?: string;
   iconOnly?: boolean;
+  isLoading?: boolean;
```

Comment:
I _think_ the `isLoading` prop should be specified in the `BaseButtonProps` or `ButtonStyles` interface instead and extended from here. 

This seems to be working though (but I'm not sure why), except for the fact that the prop isn't showing up in the props table.

Reply:
Yes, I double checked.  This prop should be removed here and added to the `ButtonStyles` interface in Button/index.tsx.  When you move it there, make sure you add a comment above it that describes the prop for the props table.  For example:

```
/**
  *  Set the button to a loading state
  * @default false
  */
  isLoading?: boolean;
```

Thanks!

Reply:
Ok, it is now in Button/index.tsx.